tests/Token.spec.ts

```diff
@@ -1,17 +1,21 @@
 ﻿import { ethers } from "hardhat";
 import { ethers as tsEthers } from "ethers";
 import { expect } from "chai";
-import { getEventData } from "./utils";
-import { Token, Token__factory } from "../build/typechain";
 
-let token: Token;
+let token: tsEthers.Contract;
 let deployer: tsEthers.Signer;
 let user: tsEthers.Wallet;
 
 describe("ERC20 Token", () => {
   before(async () => {
     deployer = (await ethers.getSigners())[0];
-    token = await new Token__factory(deployer).deploy("Token", "TKN", 18);
+
+    const REMIContract = await ethers.getContractFactory("REMI");
```

Comment:
I don't think this will work without passing a signer, will it?
```suggestion
    const REMIContract = await ethers.getContractFactory("REMI", deployer);
```

Reply:
It defaults to the first signer